### ISSUE TITLE
Finalize SEO guide

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2576]
What we're after: With 18F, we worked on an SEO guide to ensure that search engines can find FEC websites easily. Working on the content onboarding ticket, we wanted to link the SEO guide but found 18F still owned it. They've made me an editor so I can copy what's in there and put it into  our own document that FEC owns.

To do:
- [x] Move content from 18F SEO guide for FEC into the FEC's SEO guide.
- [x] Update content onboarding materials with the link to the new document.

**Comment by dorothyyeager** [comment]
I've updated the SEO guide with current examples. It's great to see that implementing some of the practices that 18F recommended improved the examples they had in their original guidance for us. Since that happened, I've edited the text to reflect some guidance and update the examples used for future content managers. We still have a ways to go, but our guide is updated and ready to help us.

https://docs.google.com/document/d/1TPPv0X7QSfXymr7RMoHxVKjX1oAvOtguK5pn7kOo7Cg/edit?usp=sharing
